.pre-commit-config.yaml

```
@@ -3,3 +3,9 @@ repos:
     rev: "v2.7.1"
     hooks:
       - id: prettier
+  - repo: https://github.com/python-jsonschema/check-jsonschema
+    rev: 0.21.0
+    hooks:
+      - id: check-jsonschema
+        files: ^modules/nf-core/[^/]+/[^/]+/meta.yml$
```

Sorry I didn't realise it was this regex. I think this should work
```suggestion
        files: ^modules/nf-core/.*/meta\.yml$
```

---

jep, also didn't see it. Tested it now and works (error messages are not so helpful, but that is not on our side...